MbDotNet/Models/Responses/Fields/ProxyResponseFields.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+using System.Net;
+using System;
+using MbDotNet.Enums;
+using MbDotNet.Models.Predicates;
+using Newtonsoft.Json.Converters;
+
+namespace MbDotNet.Models.Responses.Fields
+{
+    public class ProxyResponseFields : ResponseFields
+    {
+        [JsonProperty("to", NullValueHandling = NullValueHandling.Ignore)]
+        public Uri To { get; set; }
+
+        [JsonProperty("mode", NullValueHandling = NullValueHandling.Ignore)]
+        [JsonConverter(typeof(StringEnumConverter))]
+        public ProxyMode Mode { get; set; }
+
+        [JsonProperty("predicateGenerators", NullValueHandling = NullValueHandling.Ignore)]
+        public List<PredicateBase> PredicateGenerators { get; set; } 
```

Comment:
I'm not sure using `PredicateBase` here will work. Looking at the [documentation around predicate generators](http://www.mbtest.org/docs/api/proxies#proxy-predicate-generators), it looks like the generators always have a `matches` field. Since that seems to be identical to a regular matches predicate, we should be able to make this a `List<MatchesPredicate>` instead.

Also, please make it an `IList` instead of `List`.